src/components/HomePageHeader/homePageHeader.css

```diff
@@ -0,0 +1,102 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  background-color: #4369b2;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.title {
+  margin: 10px 15px;
+  font-size: 18px;
+  color: white;
+  font-weight: 500;
+  font-family: 'gillsans', sans-serif;
+}
+
+.header_nav a {
+  color: blue;
+}
+
+.header_logo {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  margin-left: 10px;
+}
+
+.header_RocketMissions {
+  display: flex;
+  gap: 20px;
+  margin-right: 10px;
+  border-right: 1px solid black;
+  padding-right: 10px;
+}
+
+.active {
+  text-decoration: underline;
+}
+
+.header .menu-btn {
+  display: none;
+}
+
+.mobileNavBackground {
+  background-color: #4369b2;
+}
+
+/* .mobileNavBackground:hover {
+      transition-duration: 1s;
+    background-color:#fff;;
+  } */
```

Comment:
- [ ] I don't think we are using the highlighted lines of codes. So I will suggest we remove them. Leaving them will make our codebase untidy. We can use git history if we need to revisit previous codes.

Reply:
Thanks for pointing this out, I was searching comments only with '//' but the CSS comments are different. There was also a comment in the **selectedCountryHeader.css**. This is fixed now!